index.js

```diff
@@ -117,4 +117,46 @@ app.post('/payments/create', async (request, result) => {
     }
 })
 
+// Endpoint used to edit customer
+app.post('/customers/edit', async (request, result) => {
+    const customerData = request.body;
+    console.log("Customer Edit Payload: ", customerData);
+
+    try {
+        const { name, tariffPlanId, siteId, meterNumber, hasmeter, isactive, id, customerUpdate } = customerData;
+        const { dateUpdated, customerIds, explanation, userId } = customerUpdate;
+        const airtableCustomerData = {
+            isactive,
+            hasmeter,
+            tariffPlanId,
+            siteId,
+            name,
+            meterNumber,
+        };
+        const airtableCustomerUpdate = {
+            dateUpdated,
+            customerIds,
+            explanation,
+            userId: userId[0]
+        };
+
+        await updateCustomer(id, airtableCustomerData);
+        console.log("Customer edited!");
+
+        result.status(201);
+        result.json({ status: 'OK' })
+
+        const updateId = await createCustomerUpdate(airtableCustomerUpdate);
+        console.log("Update id: ", updateId);
+        console.log("Created updates!");
+
```

Comment:
We should add error handling if the `updateCustomer` successfully goes through, but the `createCustomerUpdate` doesn't. I actually don't know how this should look (maybe we continually retry X amount of times until it succeeds?), but we can look into this together?